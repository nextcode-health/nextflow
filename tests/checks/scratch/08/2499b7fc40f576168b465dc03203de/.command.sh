#!/bin/bash -ue
echo Home: $HOME - Input: alpha zzz
