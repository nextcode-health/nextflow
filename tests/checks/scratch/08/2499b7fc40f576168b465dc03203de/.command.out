Home: /Users/olafurh - Input: alpha zzz
